src/states/common/mod.rs

```diff
@@ -25,3 +33,46 @@ pub struct Core {
     pub timer: Timer,
     pub rng: MainRng,
 }
+
+impl Core {
+    pub fn send_message_to_targets(
+        &mut self,
+        conn_infos: &[SocketAddr],
+        dg_size: usize,
+        msg: Bytes,
+    ) {
+        self.network_service
+            .send_message_to_targets(conn_infos, dg_size, msg)
+    }
+
+    pub fn send_message_to_target_later(
```

Comment:
Using "target" instead of "towards" allow us to express the distinction between sending to multiple peers ("to_targets") and single peer ("to_target") which is somehow important. Although that distinction is also expressed by the fact that one takes a collection of targets and the other just one. So maybe the whole "to_targets" / "to_target" can be removed from the name, ending with just `send_message` and `send_message_later` (or `send_message_with_delay` as you suggested).